vividus-extension-selenium/src/test/java/org/vividus/selenium/WebDriverManagerContextTests.java

```diff
@@ -82,4 +84,32 @@ void testResetParameter()
         webDriverManagerContext.reset(param);
         verify(testContext).remove(param.getContextKey());
     }
+
+    @Test
+    void testGetWithNullInitialValueSupplier()
+    {
+        WebDriverManagerParameter param = WebDriverManagerParameter.DESIRED_CAPABILITIES;
+        var  exception = assertThrows(IllegalArgumentException.class,
+                () -> webDriverManagerContext.get(param, () -> ANY_VALYE));
+        assertEquals("Initial value supplier for parameter '" + param.getContextKey() + "' is not null",
+                exception.getMessage());
+    }
+
+    @SuppressWarnings("unchecked")
+    @Test
+    void testGetWithInitialValueSupplier()
+    {
+        WebDriverManagerParameter param = WebDriverManagerParameter.DEVICE_PIXEL_RATIO;
+        webDriverManagerContext.get(param, () -> ANY_VALYE);
+        verify(testContext).get(eq(param.getContextKey()), any(Supplier.class));
```

Comment:
do not use `any` for positive tests, the logic of supplier must be checked

Reply:
done